contracts/fungible-token.aes

```diff
@@ -57,16 +63,20 @@ contract FungibleToken =
   public function allowance_from(addr: address) : int =
     Map.lookup_default((addr, Call.caller), state.allowances, 0)
 
+  // TODO check negative allowances possible
```

Comment:
The allowance is used for allowing a certain address to spend your tokens. Then the `approve` function is where you tell exactly how many tokens you give permission to the spender to spend. 

We shouldn't be able to give a negative allowance/approval at least not in the correct implementation of the fungible token standard interface. Further read [ERC20 problems](https://blog.smartdec.net/erc20-approve-issue-in-simple-words-a41aaf47bca6)

Reply:
I'll have a reas